check_dnsserials

```diff
@@ -158,10 +166,26 @@ for server in ${slaves}; do
 	if [ "x${serial_slave}" = "x${serial}" ]; then
 		verbose "OK:  ${domain} ${server} ${serial}"
 		else
-			verbose "BAD: ${domain} ${server} ${serial_slave}"
-			fails=$((fails + 1))
-			failedServers+=(${server})
-			debug "Incrementing failed states to ${fails}"
+                        if [ -n "${tolerance}" ]; then
+			        if [ $((${serial} - ${serial_slave})) -lt 0 ]; then
+			                verbose "CRITIAL: ${domain} ${server} ${serial} < ${serial_slave}"
+			                fails=$((fails + 1))
+			                failedServers+=(${server})
```

Comment:
Indentation is done here with tabs, too. Please change.

Reply:
Fixed